dpgen/generator/lib/abacus_scf.py

```diff
@@ -22,15 +22,23 @@ def make_abacus_scf_input(fp_params):
     ret += "calculation scf\n"
     for key in fp_params:
         if key == 'ntype':
+            if type(fp_params["ntype"])==str:
+                fp_params["ntype"] = int(fp_params["ntype"])
```

Comment:
how about writting
```suggestion
            fp_params["ntype"] = int(fp_params["ntype"])
```
In case someone provide fp_params["ntype"] = 2.0

Reply:
Got it. I will modify lines having the same problem.